biome.json

```
@@ -0,0 +1,179 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+  "files": {
+    "ignore": [
+      "node_modules",
+      "dist",
+      ".venv",
+      ".turbo",
+      ".nuxt",
+      ".vite",
+      ".output",
+      ".vite-ssg-temp",
+      "**/vite.config.ts.timestamp-*",
+      "package.json",
+      "scalar.config.json",
+      "**/test-results/**"
+    ]
+  },
+  "formatter": {
+    "formatWithErrors": false,
+    "ignore": [
+      ".git",
+      ".svn",
+      ".hg",
+      ".tmp",
+      "node_modules",
+      "dist",
+      "generated",
+      "venv",
+      ".venv",
+      "playwright-report",
+      ".vite-ssg-temp",
+      "pnpm-lock.yaml",
+      "fixtures"
+    ],
+    "enabled": true,
+    "lineEnding": "lf",
+    "indentWidth": 2,
```

```suggestion
    "indentWidth": 2,
    "lineWidth": 120,
```
not sure what the default is